package.json

```diff
@@ -4,22 +4,25 @@
   "private": true,
   "dependencies": {
     "@capacitor/core": "1.3.0",
-    "@ionic/react": "^5.3.1",
-    "@ionic/react-router": "^5.3.1",
+    "@ionic/react": "^6.0.0-rc.2",
+    "@ionic/react-router": "^6.0.0-rc.2",
     "@types/jest": "24.0.18",
     "@types/node": "12.7.5",
     "@types/react": "^16.9.43",
     "@types/react-dom": "^16.9.8",
     "@types/react-router": "^5.1.8",
     "@types/react-router-dom": "^5.1.5",
+    "date-fns": "^2.25.0",
     "ionicons": "^5.0.1",
+    "mini-create-react-context": "^0.4.1",
     "node-sass": "^4.13.0",
```

Comment:
Can we migrate this to `sass` instead of `node-sass`? That version listed here is not compatible with my current environment. Probably easier to avoid the C++ bindings.

Reply:
Done 👍